docs/se/guides/tracing.adoc

```diff
@@ -281,7 +305,17 @@ app:
   greeting: "Hello From SE-2"
 
 tracing:
-  service: "helidon-se-2"
+  service: helidon-se-2
+  protocol: http
+  port: 14250
+  path: /api/traces
+  tags:
+    env: development
+  enabled: true
+  sampler-type: "const"
+  sampler-param: 1
+  log-spans: true
+  propagation: b3
```

Comment:
It's very good to use config settings that will make tracing work easily as a user experiments with it. (The param = 1 for example, IIRC, makes sure traces are flushed right away so they appear in the back-end system with minimal buffering or delay.)

That said, maybe we add a brief sentence explaining that and advising users to use different settings for production servers for performance reasons.

Reply:
Fixed :)